website/blog/2019-05-13-the-new-docker-image-verdaccio-4.md

```diff
@@ -0,0 +1,60 @@
+---
+author: Juan Picado
+authorURL: https://twitter.com/jotadeveloper
+authorFBID: 1122901551
+title: The new docker image for Verdaccio 4
+---
+
+Docker has been for Verdaccio a key part of his success, at the time of this writing we have more than 4 millions and growing rapidly. The image provides an easy installation and integration with other tools, as Kubernetes, or composing them with any other Docker image. 
+
+This article aims to describe what have changed and all the improvements and benefits you will enjoy migrating to latest version.
+
+## What’s new?
+
+
+### Keep it small
+
+The new image is three times smaller than the previous, around 150 MB instead 500MB, that’s all. We have optimized using [Multi Stage build](https://medium.com/capital-one-tech/multi-stage-builds-and-dockerfile-b5866d9e2f84), this allow us to exclude dependencies and assets are not required for the runtime.  
+
+### Environment Variables
+
+To avoid mistakes we have renamed all environment variables being prefixed with VERDACCIO_. This will avoid future collisions and better understanding of the origin and usage of the variable. Here the full list of the new list of variables available in the new image.
+
+[table]
+
+### Support Arbitrary User IDs
+
+The previous image runs the container with the verdaccio user and group by default, being the UID created randomly within the image. Some users were experimenting issues since some environment requires usage of arbitrary user IDs for security reasons, for such reason we have introduced the environment variable `VERDACCIO_USER_ID` that allows that. 
```

Comment:
```suggestion
The previous image runs the container with the verdaccio user and group by default, being the UID created randomly within the image. Some users were experiencing issues since some environments require the usage of custom user IDs for security reasons. To support this, we have introduced the environment variable `VERDACCIO_USER_ID`.
```

Assumption: It is actually not literally *arbitrary* user IDs but *custom* ones.